src/domains/user/accounts.routes.integration.spec.js

```diff
@@ -18,10 +18,122 @@ describeDbTestSuite('accountsRouter', () => {
         .post('/signup')
         .set('Content-type', 'application/json')
         .send(createAccountBody);
+      const { message, token } = response.body;
       expect(response.body).toEqual({
-        message: 'account created',
+        message,
+        token,
       });
       expect(response.status).toEqual(201);
     });
+
+    test('it should not create an account without email', async () => {
+      const createAccountBody = {
+        password: '0PasSwoRd12',
+      };
+
+      const response = await request(app)
+        .post('/signup')
+        .set('Content-type', 'application/json')
+        .send(createAccountBody);
+      const { message } = response.body;
+      expect(response.body).toEqual({
+        message,
+      });
+      expect(response.status).toEqual(400);
+    });
+
+    test('it should not create an account without password', async () => {
+      const createAccountBody = {
+        email: 'email@example.com',
+      };
+
+      const response = await request(app)
+        .post('/signup')
+        .set('Content-type', 'application/json')
+        .send(createAccountBody);
+      const { message } = response.body;
+      expect(response.body).toEqual({
+        message,
+      });
+      expect(response.status).toEqual(400);
+    });
+
+    test('it should not create an account with invalid email', async () => {
+      const createAccountBody = {
+        email: 'emailexample.com',
+        password: '0PasSwoRd12',
+      };
+
+      const response = await request(app)
+        .post('/signup')
+        .set('Content-type', 'application/json')
+        .send(createAccountBody);
+      const { message } = response.body;
+      expect(response.body).toEqual({
+        message,
+      });
+      expect(response.status).toEqual(400);
+    });
+
+    test('it should not create an account with short password', async () => {
+      const createAccountBody = {
+        email: 'email@example.com',
+        password: '0Pas',
+      };
+
+      const response = await request(app)
+        .post('/signup')
+        .set('Content-type', 'application/json')
+        .send(createAccountBody);
+      const { message } = response.body;
+      expect(response.body).toEqual({
+        message,
+      });
+      expect(response.status).toEqual(400);
+    });
+
+    test('It should return invalid when login with wrong credential', async () => {
+      const credentials = {
+        email: 'wrong@example.com',
+        password: 'wrongpass',
+      };
+
+      const response = await request(app)
+        .post('/login')
+        .set('Content-type', 'application/json')
+        .send(credentials);
+      const { message } = response.body;
+      expect(response.body).toEqual({
```

Comment:
your tests are not concrete enough. you should test for what you are expecting from the application against what the app is returning. 

it would make more sense if your test was this way

`
const expectedMessage = 'registration successful';
expect(req.body.message).toEqual(expectedMessage);
`
this way we can be sure that our app behaves as expected when the tests pass.